README.md

```diff
@@ -102,6 +103,33 @@ The following is a brief explanation of each available command.
 * `validate-config`: Validates the `.codeclimate.yml` file in the current working directory.
 * `version`: Displays the current version of the Code Climate CLI.
 
+## Debugging
```

Comment:
I would call this section something else (maybe Environment Variables?). Only `CODECLIMATE_DEBUG` has to do with debugging, the other two don't.